### ISSUE TITLE
Betting in general Bug

### ISSUE THREAD

**Issue by thekealganon** [saltbot-org/saltbot#453]
There is a bug I believe where Saltbot places a bet and the green arrow triggers on the screen but sometimes the bet won't go through. It seems to be fixed partially when I change the AI setting for example switching from scientist to cowboy etc. I uninstalled and reinstalled the program and it does not seem to fix the problem. What I have noticed however is that even when I manually insert a bet on the SB website, the bet will sometimes not go through as well. I tested this without Saltbot activated on the site and everything works fine. The bug seemingly only appears when Saltbot is activated on the site.

In short, the SaltBot bugs out the Saltybet website somewhat with how the bets work.

Tested with all Adblockers disabled and then subsequently a vanilla chrome. The issue still persists.

**Comment by Vehdrehl** [comment]
I can confirm the above. Betting isn't being places properly

**Comment by reconman** [comment]
I currently can't reproduce it.

**Comment by volke0** [comment]
Yeah, I don't know how to reproduce it either, but I can confirm it happens to me as well and is extremely annoying to deal with.  Refreshing the page doesn't help at all either.

**Comment by Vehdrehl** [comment]
I just noticed it stuck at a certain point and wouldn't place bets anymore. I don't know how to reproduce it either as I can't reset it to actively bet.

**Comment by thekealganon** [comment]
From what I've learned, it has something to do with the green check mark in between the red and blue option. If the green check mark doesn't appear, the bet won't go through. However, you are still able to press the buttons and the arrow will point to the color you choose.

**Comment by volke0** [comment]
Hey all!  Although I don't use this bot at the moment, I found something quite interesting in another Salty Bet bot's FAQ:

"Q: Help! The bot stopped betting!

A: A recent change in Chrome causes the bot to break if the SaltyBet tab isn't active. You can fix this by keeping the SaltyBet tab active, or you can go to chrome://flags and then disable "Throttle Javascript timers in background"."

This may actually apply to this bot as well, although I am quite unsure since I haven't tested it.  Let me know if you do!

**Comment by Vehdrehl** [comment]
Just came back to this as it appears the bot isn't betting without window focus. The flag mentioned above no longer exists, at least not where stated above. Would be great to have the bot better in the background.

**Comment by volke0** [comment]
The flag still exists, you just need to enable #temporary-unexpire-flags-m94 and #temporary-unexpire-flags-m95, and to then disable the throttling of the Javascript for inactive tabs is the #intensive-wake-up-throttling. This is what I mentioned before, but this is the actual name of the flag so when you search for it you will the correct one.
